compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -29,6 +29,17 @@ def AMDAIEAssignChannels :
   let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIEAssignChannelsPass()";
 }
 
+def AMDAIEAssignConnectionTypes :
+    Pass<"iree-amdaie-assign-connection-types", ""> {
```

Comment:
I don't see corresponding lit test file for this pass under `Transforms/test`.

Reply:
brutal